

RRTStarPlanner:
  # rrt star parameters
  goal_tolerance: 0.2
  radius: 0.5
  epsilon: 0.2
  max_num_nodes: 10000
  min_num_nodes: 1000
  # WOA Parameters
  sampling_radius: 0.1
  max_iterations: 1000
  num_agents: 5
  spiral_shape: 0.4